phpmyfaq/admin/assets/scss/style.scss

```
@@ -31,6 +31,10 @@
 // Import Mixins
 @import 'mixins.scss';
 
+// Semantic design tokens — must come before the global admin styles so
+// component rules can consume the token values.
+@import './variables/tokens.scss';
```

Stylelint (17.11.0)

[error] Unexpected extension ".scss" in `@import` (scss/load-partial-extension)

(scss/load-partial-extension)